README.md

```diff
@@ -49,9 +49,9 @@ To perform the tests locally you'll need the `API_KEY` and `HOST_URL` environmen
 
 | Name                   | Description                                                                 | Required | Default Value |
 |------------------------|-----------------------------------------------------------------------------|----------|---------------|
-| `apiKey`            | The Pinecone api key.                          | Yes      |               |
+| `apiKey`            | The Pinecone API key.                          | Yes      |               |
 | `host`            | The Pinecone index host.                          | Yes      |               |
-| `namespace`            | The Pinecone namespace to target. Defaults to the default Pinecone namespace                           | No      |               |
+| `namespace`            | The Pinecone namespace to target. It can contain a [Go template](https://pkg.go.dev/text/template) that will be executed for each record to determine the namespace. By default, the namespace will come from the `opencdc.collection` record metadata field. | No      | `opencdc.collection`              |
```

Comment:
```suggestion
| `namespace`            | The Pinecone namespace to target. It can contain a [Go template](https://pkg.go.dev/text/template) that will be executed for each record to determine the namespace. By default, the namespace will come from the `opencdc.collection` record metadata field. | No      | `{{ index .Metadata "opencdc.collection" }}`              |
```